lib/rubocop/cop/rspec/let_before_examples.rb

```diff
@@ -64,6 +74,39 @@ def check_let_declarations(node)
             end
           end
         end
+
+        def find_first_example(node)
+          node.parent.children.find { |sibling| example_or_group?(sibling) }
+        end
+
+        def node_range(node)
```

Comment:
I generally prefer if methods always call "down" in the source file, if possible. Since this method is called by `#node_range_with_surrounding_space` and `#source`, it should be moved below them.